src/js/common/schemaform/widgets/RadioWidget.jsx

```diff
@@ -28,7 +28,7 @@ export default function RadioWidget({
               autoComplete="false"
               checked={checked}
               id={`${id}_${i}`}
-              name={`${id}_${i}`}
```

Comment:
Oh, wow. Using controlled components seems to have kept us from ever showing multiple radio buttons selected at once, even though the html allowed it.

Reply:
Heh, huzzah for being accidentally correct!  🎉

Reply:
Hm, fascinating find.